tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py{27,34,35}-django19
+skipsdist = True
+
+[tox:travis]
+2.7 = py27
+3.4 = py34
+3.5 = py35
+
+[testenv]
+passenv = *
+deps = -rrequirements.txt
```

Comment:
should there be a space here?

Reply:
Nope – it works without and internally it's passing to `popen` or something that takes a list and converts them to args. Basically, everything after the space is ignored – nb I copied this from polling stations :)

Reply:
Now you mention it, I actually remember getting confused by this before. I expect it will probably happen again..